src/Microsoft.Identity.Client/Platforms/Shared/NetStdCore/NetStandardCryptographyManager.cs

```diff
@@ -0,0 +1,74 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Security.Cryptography;
+using System.Security.Cryptography.X509Certificates;
+using System.Text;
+using Microsoft.Identity.Client.Core;
+using Microsoft.Identity.Client.PlatformsCommon.Interfaces;
+using Microsoft.Identity.Client.Utils;
+
+namespace Microsoft.Identity.Client.Platforms.Shared.NetStdCore
```

Comment:
mysterious namespace

Aside: why introduce the shared namespace?

Reply:
Well we try to keep the namespaces aligned with the directories. And it makes sense to have  Shared/NetStdCore, Shared/Apple and Shared/Desktop, otherwise the same class would be in 2 places.